hack/ginkgo-e2e.sh

```diff
@@ -162,6 +162,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --cluster-monitoring-mode="${KUBE_ENABLE_CLUSTER_MONITORING:-standalone}" \
   --prometheus-monitoring="${KUBE_ENABLE_PROMETHEUS_MONITORING:-false}" \
   --dns-domain="${KUBE_DNS_DOMAIN:-cluster.local}" \
+  --ginkgo.slowSpecThreshold="${GINKGO_SLOW_SPEC_THRESHOLD:-5}" \
```

Comment:
Can we set the default value more appropriately? I'm pretty sure that of our e2e tests complete in 5s

Reply:
@fejta Sure... I am not sure about an optimal value as I am pretty new here. Kindly let me know what would be a reasonable value for the slowSpecThreshold.